packages/vuetify/src/components/VFileInput/VFileInput.ts

```diff
@@ -115,7 +116,7 @@ export default VTextField.extend({
       return this.isDirty
     },
     isMultiple (): boolean {
-      return this.$attrs.hasOwnProperty('multiple')
+      return this.multiple
```

Comment:
The reporting issue had a one liner to solve this issue, is there a reason why we can't use that instead?

```return this.$attrs.hasOwnProperty('multiple') && this.$attrs.multiple !== false```